mne/commands/tests/test_commands.py

```diff
@@ -202,4 +203,37 @@ def test_watershed_bem():
         mne_watershed_bem.run()
 
 
+@slow_test
+@requires_mne
+@requires_freesurfer
+@sample.requires_sample_data
+def test_flash_bem():
```

Comment:
grrr. Anyone else can reproduce? I used to get this error but now it seems
to work for me